src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2618,26 +2618,33 @@ A complete example of Producer / Consumer, _ServiceDeployerConsumer_ can be down
-That is, it allows a job to expose specific signals (e.g., `soft_stop` to properly finish a job) that can be triggered by users from the *Workflow Execution portal*.
+That is, it allows a job to expose specific signals (e.g., `soft_stop` to properly finish a job) that can be triggered by users from the *Workflow Execution portal* or *Scheduler portal*.
 You can try the <<_examples_2,examples>> below to better understand how it works.
 
 In summary and for simple use, just go to <<_proactive_studio, ProActive Studio>> and use the 2 predefined templates in the Controls menu: *Check_For_Signals* or *Wait_For_Signals*.
-Fill up the Task Variable *SIGNALS*, and use those templates in your own workflow to be able to receive input from, for instance, the Workflow Execution portal.
+Fill up the Task Variable *SIGNALS*, and use those templates in your own workflow to be able to receive input from, for instance, the Workflow Execution portal or Scheduler portal.
 
 At a more detailed level and advanced use, the Signal API allows jobs and users to exchange signals using a communication channel provided by the <<_task_synchronization_api,synchronization API>>.
 The typical usage scenario of the Signal API is the following:
 
-  . A job declares to be ready to handle one or more signals. The signals are then displayed in the Workflow Execution portal (e.g., display the `soft_stop` signal).
+  . Signals are defined by their name and, optionally, a list of variables. Variables attached to a signal have the same structure as <<_workflow_variables>> but are declared exclusively using the <<_task_signal_api,Signal API>>.
+
+  . If the signal has variables, then a new window will appear where the user can set the variables values and then send the signal with variables to the job. If the signal does not have variables then the window will not appear.
+    Adding variables to a signal allows to enhance the manual control of the workflow execution.
+    After receiving the signal with attached variables, the workflow handles the provided input data properly.
+
+  . A job declares to be ready to handle one or more signals that can have attached variables. The signals are then displayed in the Workflow Execution portal or Scheduler portal(e.g., display the `soft_stop` signal).
 
   . The job waits to receive these signals, or check for their reception regularly.
 
-  . A user clicks on the signal displayed in the Workflow Execution portal (e.g., click on the `soft_stop` signal), which sends this signal to the job.
+  . A user clicks on the signal displayed in the Workflow Execution portal or Scheduler portal (e.g., click on the `soft_stop` signal), which sends this signal to the job.
 
   . At the signal reception, the job executes the corresponding behavior (defined in the workflow).
+    When the signal has attached variables, the workflow will receive the signal only if the received variables are compatible with the variables set at the ready instruction.
```

Comment:
"the workflow will receive the signal only if the received variables are compatible with the variables set at the ready instruction"
This is not clear. I would rather write
When the signal has attached variables, variables must be valid according to their model specification (see <refrence to variables model>). Sending a signal will not be possible unless they are validated.